common/src/test/java/org/apache/uniffle/common/security/HadoopSecurityContextTest.java

```diff
@@ -119,5 +117,28 @@ public void testCreateIllegalContext() throws Exception {
     } catch (Exception e) {
       assertTrue(e.getMessage().contains("refreshIntervalSec must be not negative"));
     }
+
+    // case4: lack krb5 conf, should throw exception
+    String krbConfFilePath = System.getProperty("java.security.krb5.conf");
+    System.clearProperty("java.security.krb5.conf");
+    try {
+      HadoopSecurityContext context = new HadoopSecurityContext(
+          null,
+          kerberizedHdfs.getHdfsKeytab(),
+          kerberizedHdfs.getHdfsPrincipal(),
+          100
+      );
+      fail();
+    } catch (Exception e) {
+      assertTrue(e.getMessage().contains("Cannot locate KDC"));
+    }
+
+    // case5: After setting the krb5 conf, it should pass
+    HadoopSecurityContext context = new HadoopSecurityContext(
+        krbConfFilePath,
+        kerberizedHdfs.getHdfsKeytab(),
+        kerberizedHdfs.getHdfsPrincipal(),
+        100
+    );
```

Comment:
Should we recover the System properties to avoid that this operation affect other tests?

Reply:
The System properties will been reset in HadoopSecurityContext. But I think explicitly recovering properties is also OK in this case.